sdk/python/tests/test_feature_store.py

```diff
@@ -158,14 +158,16 @@ def test_apply_entity_integration(test_feature_store):
 @pytest.mark.parametrize(
     "test_feature_store", [lazy_fixture("feature_store_with_local_registry")],
 )
-def test_apply_feature_view_success(test_feature_store):
+@pytest.mark.parametrize("protocol", ["file", "s3"])
+def test_apply_feature_view_success(test_feature_store, protocol):
```

Comment:
What does this test actually test with the new change?

Reply:
In this case, it only checks the feature store definition apply, which will be important if someone will change protos or data_source code.